lib/Buzz/Client/AbstractCurl.php

```diff
@@ -14,12 +14,14 @@
 abstract class AbstractCurl extends AbstractClient
 {
     protected $options = array();
+    protected static $headersSize = 0;
+    protected static $headers = array();
```

Comment:
Actually not, and I don't like it so much, but I've tried to respect as much as possible the current project architecture and I think that handle the headers in other class would be odd.